build-configs/aschaffenburg/assets/location-marker.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 150">
```

Comment:
Do we require this with flow? Because it won't be used anyway as there won't be a landing page for aschaffenburg.

Reply:
removed it :)